hub/core/tensor.py

```diff
@@ -793,11 +807,60 @@ def data(self, aslist: bool = False) -> Any:
                 "value": self.numpy(aslist=aslist),
                 "sample_info": self.sample_info or {},
             }
+        elif htype == "point_cloud":
+            full_arr = self.chunk_engine.numpy(
```

Comment:
use self.numpy instead

Reply:
yes but instead of calling self.chunk_engine.numpy on this line, call self.numpy

Reply:
But it will return only xyz, but I want to return bigger array